readme.txt

```diff
@@ -33,6 +33,15 @@ Step 2:
 
 Install the plugin and activate it.
 
+Step 3:
+
+WordPress has a minimum password length of 1, and passwords are hashed using an MD5-based algorithm instead of a real password hashing algorithm (such as bcrypt, scrypt, or PBKDF2).
+
+We recommend addressing those issues, but we don't recommend a particular plugin. However if you think WordPress should have these (basic) features, you should register your interest on the tickets:
+
+- https://core.trac.wordpress.org/ticket/21022
```

Comment:
Is there any reason we can't mention our own bcrypt plugin? https://github.com/dxw/wp_bcrypt. Not to push it, but just as helpful info.

Reply:
I think it's probably best not to recommend anything that we're not actually using ourselves.

That plugin hasn't been updated in over 3 years. It may not even work with the current version of WordPress.